packages/plugins/rubygem-puppetdb_foreman/rubygem-puppetdb_foreman.spec

```diff
@@ -1,79 +1,52 @@
-# This package contains macros that provide functionality relating to
-# Software Collections. These macros are not used in default
-# Fedora builds, and should not be blindly copied or enabled.
-# Specifically, the "scl" macro must not be defined in official Fedora
-# builds. For more information, see:
-# http://docs.fedoraproject.org/en-US/Fedora_Contributor_Documentation
-# /1/html/Software_Collections_Guide/index.html
-
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: foreman_plugin
 %global gem_name puppetdb_foreman
-
-%global foreman_dir /usr/share/foreman
-%global foreman_bundlerd_dir %{foreman_dir}/bundler.d
-
-Summary:    Foreman plugin to interact with PuppetDB
-Name:       %{?scl_prefix}rubygem-%{gem_name}
-Version:    5.0.0
-Release:    4%{?foremandist}%{?dist}
-Group:      Applications/System
-License:    GPLv3
-URL:        https://github.com/theforeman/puppetdb_foreman
-Source0:    https://rubygems.org/gems/%{gem_name}-%{version}.gem
-
-Requires:   foreman >= 1.17.0
-
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby
-Requires: %{?scl_prefix_ruby}rubygems
-
-BuildRequires: foreman-plugin >= 1.17.0
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
-
+%global plugin_name puppetdb_foreman
+%global foreman_min_version 3.1
+
+Name: rubygem-%{gem_name}
+Version: 6.0.0
+Release: 1%{?foremandist}%{?dist}
+Summary: This is a Foreman plugin to interact with PuppetDB
+License: GPLv3
+URL: https://www.github.com/theforeman/puppetdb_foreman
+Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
+
+# start specfile generated dependencies
+Requires: foreman >= %{foreman_min_version}
+Requires: ruby
+BuildRequires: foreman-plugin >= %{foreman_min_version}
+Requires: ruby
+BuildRequires: ruby
+BuildRequires: rubygems-devel
 BuildArch: noarch
-
-Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
-Provides: foreman-plugin-puppetdb
-%{?scl:Obsoletes: ruby193-rubygem-%{gem_name}}
+Provides: foreman-plugin-%{plugin_name} = %{version}
+# end specfile generated dependencies
 
 %description
-This is a foreman plugin to interact with PuppetDB through callbacks
-and proxy the performance dashboard to Foreman.
+Disable hosts on PuppetDB after they are deleted or built in Foreman. Follow
+https://github.com/theforeman/puppetdb_foreman and raise an issue/submit a
+pull request if you need extra functionality. You can also find some help in
+theforeman IRC channel on Freenode.
```

Comment:
```suggestion
theforeman IRC channel on libera.chat.
```
right? ;)

Reply:
https://github.com/theforeman/puppetdb_foreman/pull/76

Reply:
:+1: I will update the description manually when the plugin decided on a new text (subscribed to the pull request so I hopefully see it).

Reply:
Pushed it now and hoping to see some green! ;-)